packages/itwin/reports-config-widget/src/widget/components/SearchBar.tsx

```diff
@@ -20,29 +20,17 @@ export const SearchBar = ({
   disabled,
 }: SearchBarProps) => {
   const [searchBarOpen, setSearchBarOpened] = useState<boolean>(false);
-  const [searchBarClosing, setSearchBarClosing] = useState<boolean>(false);
 
   return searchBarOpen || searchValue ? (
-    <div className="rcw-search-button"
-      style={{
-        animation: searchBarClosing ? "rcw-shrink .5s" : "rcw-expand .5s",
-      }}
-      onAnimationEnd={() => {
-        if (searchBarClosing) {
-          setSearchBarClosing(false);
-          setSearchBarOpened(false);
-          setSearchValue("");
-        }
-      }}
-    >
+    <div className="rcw-search-button">
```

Comment:
Looks like there is a visual change (removed animation) and functional change (removed search value reset). Why is this not in a separate PR? Why such change even needs to be made?

Reply:
I will leave this to another PR. I was having issues with the animation.